cards.js

```diff
@@ -16,8 +16,124 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
+function buildDeck(suits, values){
+  var deck = [];
+  for(i in suits) {
+    for(j in values) {
+      deck.push({suit: suits[i], value: values[j]});
+    }
+  }
+  return deck;
+}
+
+newDeck = buildDeck(suits, values);
+newDeck = shuffle(newDeck);
+
+var namePlayers = function() {
+  var player1 = "Joe";
+  var player2 = "Frederico";
+  return [player1, player2];
+}
+
+
+var highCard = function(deck, namePlayers, values) {
+  players = namePlayers();
+  playerCards = [];
+  for(i in players) {
+    playerCards.push({player: players[i], card: deck[i]});
+    console.log(players[i] + "'s card is the " + playerCards[i]["card"]["value"] + " of " + playerCards[i]["card"]["suit"]);
+  };
+  if(values.indexOf(playerCards[0].card.value) > values.indexOf(playerCards[1].card.value)) {
+    console.log(players[0] + " is the winner!");
+  }
+  else if(values.indexOf(playerCards[0].card.value) < values.indexOf(playerCards[1].card.value)) {
+    console.log(players[1] + " is the winner!");
+  }
+  else if(values.indexOf(playerCards[0].card.value) === values.indexOf(playerCards[1].card.value)) {
+    console.log("It's a tie!")
+  }
+}
+
+highCard(newDeck, namePlayers, values);
 
-// Put some code in here!
+var joesDeck = function(deck) {
+  joesDeck = [];
+  for(i = 0; i < deck.length; i = i + 2) {
+    joesDeck.push(deck[i]);
+  }
+  return joesDeck;
+}
```

Comment:
on lines 60 and 64, you may want to choose a different name for `joesDeck`. Maybe something like `playersDeck`. At the end of the day the function is returning a deck of cards and you already have a buildDeck/shuffleDeck method. You could just say `var joesDeck = shuffleDeck(buildDeck)` or something like that(not sure exactly what the code is for shuffling) That way you don't have to create two more functions for something that essentially already exists.